package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-backpack",
-  "version": "2.0.3",
+  "version": "2.0.4-beta.0",
```

Comment:
Do we want to release a beta?

Reply:
That is ok right now as I released one to test if this works, when it goes into master and we run `npm version major` this will be updated before we publish 😄 but happy to revert if necessary

Reply:
I've reverted just for audit purposes 😄